planner_logic_plugin_name: "moveit_hybrid_planning/MotionCompensation"
robot_description: "robot_description"
planning_group: "arm"
# TODO: load from yaml
robot_description_kinematics:
  arm:
    kinematics_solver: pick_ik/PickIkPlugin
    kinematics_solver_timeout: 0.05
    kinematics_solver_attempts: 3
    mode: global
    position_scale: 1.0
    rotation_scale: 0.5
    position_threshold: 0.001
    orientation_threshold: 0.01
    cost_threshold: 0.001
    minimal_displacement_weight: 0.0
    gd_step_size: 0.0001